docs/jwst/pipeline/calwebb_coron3.rst

```diff
@@ -17,24 +17,25 @@ below.
 +---------------------------------------------------+
 | calwebb_coron3                                    |
 +===================================================+
+| :ref:`outlier_detection <outlier_detection_step>` |
++---------------------------------------------------+
 | :ref:`stack_refs <stack_refs_step>`               |
 +---------------------------------------------------+
 | :ref:`align_refs <align_refs_step>`               |
 +---------------------------------------------------+
 | :ref:`klip <klip_step>`                           |
 +---------------------------------------------------+
-| :ref:`outlier_detection <outlier_detection_step>` |
-+---------------------------------------------------+
 | :ref:`resample <resample_step>`                   |
 +---------------------------------------------------+
 
 The high-level processing provided by these steps is:
 
-1) Accumulate all reference PSF images into a single product
-2) Align every PSF image to every science target image
-3) Compute an optimal PSF fit and subtract it from every science target image
-4) Compare the PSF-subtracted target images against one another to identify outliers
-5) Combine the PSF-subtracted and CR-flagged images into a single resampled image
+1) CR-flag all PSF and science target exposures
+2) Accumulate all reference PSF images into a single product
+3) Align every PSF image to every science target image
+4) Compute an optimal PSF fit and subtract it from every science target image
+5) Compare the PSF-subtracted target images against one another to identify outliers
```

Comment:
This line needs to be removed - it's replaced by item 1 (we still have a total of 5 steps, not 6).